src/components/app/routes/data/utils.js

```diff
@@ -345,13 +346,21 @@ export async function ensureActiveEnterpriseCustomerUser({
     };
   }
 
+  if (staffEnterpriseCustomer) {
```

Comment:
Ensures the redirect to active ECU does NOT occur for staff users, to support masquerading.